provider.tf

```diff
@@ -0,0 +1,16 @@
+#-------------------------------------------------------------------------------------------------------------------------------
+# Terraform Plugin - AWS cloud provider
+#-------------------------------------------------------------------------------------------------------------------------------
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.0"
+    }
+  }
+}
+
+# Configure the AWS Provider
+provider "aws" {
+  region = "eu-west-2"
+}
```

Comment:
could you please, add a newline at the end of file.

Reply:
Added the trailing line to provider.tf